kroxylicious-filters/kroxylicious-multitenant/src/test/java/io/kroxylicious/proxy/filter/multitenant/MultiTenantFilterFactoryTest.java

```diff
@@ -19,8 +20,8 @@ class MultiTenantFilterFactoryTest {
     @Test
     void testGetInstance() {
         MultiTenantTransformationFilterFactory factory = new MultiTenantTransformationFilterFactory();
-        Filter filter = factory.createFilter(Mockito.mock(FilterFactoryContext.class), null);
+        Filter filter = factory.createFilter(Mockito.mock(FilterFactoryContext.class), Mockito.mock(MultiTenantConfig.class));
```

Comment:
We should probably have an additional test which asserts the behaviour when `null` is passed in.

Reply:
I strengthened the tests too.

Reply:
I'm finding the diff impossible to read on my phone but go ahead and merge.